src/api/spec/controllers/webui/package_controller_spec.rb

```diff
@@ -901,8 +901,8 @@ def remove_file_post
         post :save_meta, params: { project: source_project, package: 'blah', meta: valid_meta }
       end
 
-      it { expect(flash[:error]).to eq("Error while saving the Meta file: Package doesn't exists in that project..") }
-      it { expect(response).to have_http_status(:bad_request) }
+      it { expect(flash[:error]).to eq("Package \"blah\" not found in project #{source_project.name.inspect}") }
```

Comment:
Why `.inspect`?

Reply:
ugly hack to make my test pass because of the quotes :pineapple: :pineapple:

Reply:
`in project '#{source_project.name}'"` doesn't work?

Reply:
or even just #{source_project}